sphinx/util/logging.py

```diff
@@ -161,12 +161,12 @@ def warning(  # type: ignore[override]
     ) -> None:
         """Log a sphinx warning.
 
-        It is recommended to always include a ``type`` and ``subtype`` for warnings.
-        These can be displayed to the user, using :confval:`show_warning_types`,
-        and allow for suppression of specific warning, using :confval:`suppress_warnings`.
+        It is recommended to include a ``type`` and ``subtype`` for warnings,
+        which can be displayed to the user using :confval:`show_warning_types`
```

Comment:
```suggestion
        It is recommended to include a ``type`` and ``subtype`` for warnings as
        these can be displayed to the user using :confval:`show_warning_types`
```

i think what you had before was actually better

Reply:
Probably lol, I'm not so bothered about the wording TBH, just as long as the general message gets across 😄